src/routes/plans/[id]/+page.ts

```diff
@@ -44,12 +44,14 @@ export const load: PageLoad = async ({ parent, params, url }) => {
 
       const initialActivityTypes = await effects.getActivityTypes(initialPlan.model_id, user);
       const initialResourceTypes = await effects.getResourceTypes(initialPlan.model_id, user, 20);
+      const initialExternalEventTypes = await effects.getExternalEventTypes(planId, user);
       const initialPlanTags = await effects.getPlanTags(initialPlan.id, user);
       const initialView = await effects.getView(
         url.searchParams,
         user,
         initialActivityTypes,
         initialResourceTypes,
+        initialExternalEventTypes,
```

Comment:
Is this used?

Reply:
This is still (conditionally) used - if your plan has at least one derivation group linked to it (and therefore has at least one external event type linked to it), an 'External Events' row will be created with these `initialExternalEventTypes` when you load the plan. See `generateDefaultView` in `utilities/view.ts` for the logic!